modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -428,61 +428,69 @@ export default {
       transition: opacity 0.2s ease;
       pointer-events: none;
     }
+
     &:before {
       top: 0;
     }
     &:after {
       bottom: 0;
     }
     &.apos-highlight {
-      z-index: $z-index-widget-highlight;
       &:before, &:after {
         opacity: 0.4;
       }
     }
     &.apos-focus {
-      z-index: $z-index-widget-focus;
       &:before, &:after {
         opacity: 1;
         border-top: 1px solid var(--a-primary);
```

Comment:
Not an issue to hold this PR for, but have we bogarted the entire "a" namespace for CSS variables? Or is it just a SASS thing?

Reply:
we're bogarting it.